examples/experimental/slpk-in-browser/tsconfig.json

```diff
@@ -1,27 +1,20 @@
 {
   "compilerOptions": {
-    "target": "es2020",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
-    "allowJs": true,
-    "checkJs": false,
-    "skipLibCheck": true,
-    "esModuleInterop": true,
+    "target": "ESNext",
```

Comment:
Do we really need all these? I am not convinced.

Reply:
Done